src/main/resources/default/assets/common/scripts/slider.js.pasta

```diff
@@ -6,16 +6,30 @@
      * Before calling this function, the slider already has to exist in the form of a html node containing all slides
      * as 'div' child nodes.
      *
-     * @param target  css selector selecting the outer html node containing all slides as child nodes
-     * @param options optional object specifying the behaviour of the slider (see constants for possible values):
-     *                 - animation:  Time in ms between sliding (can be undefined for no animation)
-     *                 - columns:    Number of columns to show/slide (defaults to 1, blending only works for one column)
-     *                 - columnsMd:  Number of columns to show/slide on medium screens
-     *                 - columnsSm:  Number of columns to show/slide on small screens
-     *                 - interrupt:  What actions interrupt the animation
-     *                 - transition: Behaviour of the animation
-     *                 - arrows:     Whether previous and next arrows should be shown and the location of them
-     *                 - dots:       Whether dots (showing currently selected slide) should be shown
+     * @param {string} target
+     *        CSS selector for the outer html node containing all slides as child nodes
+     * @param {Object} [options]
+     *        configuration specifying the behaviour of the slider
+     * @param {number} [options.animation]
+     *        time in milliseconds before switching to the next view automatically, no automatic sliding if `undefined`
+     * @param {('fade'|'slide')} [options.transition]
+     *        transition animation style when sliding, defaults to no animation when `undefined`; fading only works if
+     *        `columns` is set to 1
+     * @param {number} [options.columns=1]
+     *        number of columns to show at a time, by default 1
+     * @param {number} [options.columnsMd=columns]
+     *        number of columns to show at a time on medium-sized screens, by default equal to `columns`; must not be
+     *        larger than `columns`
+     * @param {number} [options.columnsSm=columnsMd]
+     *        number of columns to show at a time on small-sized screens, by default equal to `columnsMd`; must not be
+     *        larger than `columnsMd`
+     * @param {boolean} [options.interrupt=false]
+     *        if **true,** automatic sliding will be suspended while the mouse pointer hovers over the slider
+     * @param {boolean} [options.arrows=false]
+     *        if **true,** clickable arrows for the previous and next views are rendered on the left and on the right of
+     *        the content, respectively
+     * @param {boolean} [options.dots]
+     *        if **true,** clickable dots showing the currently selected view are rendered below the content
```

Comment:
Was the place of the comma intentionally?

Reply:
Or should it be **true**, ...

Reply:
I usually put the comma into the emphasis, but we can also change this